org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -36,14 +38,14 @@
             <unit id="org.eclipse.xtext.sdk.feature.group" version="0.0.0"/>
             <repository location="http://download.eclipse.org/releases/2019-06/"/>
         </location>
-        <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
-            <repository location="https://github.com/akaroml/lsp4j-latest/raw/master/p2/"/>
-            <unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.8.0.v20190701-0317"/>
-        </location>
         <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
             <unit id="org.jboss.tools.maven.apt.core" version="0.0.0"/>
             <repository location="http://download.jboss.org/jbosstools/updates/m2e-extensions/m2e-apt/1.5.2-2018-12-24_15-46-05-H18/"/>
         </location>
+	    <location includeAllPlatforms="false" includeConfigurePhase="false" includeMode="planner" includeSource="true" type="InstallableUnit">
+		    <repository location="http://services.typefox.io/open-source/jenkins/job/lsp4j/job/master/lastSuccessfulBuild/artifact/build/p2-repository/"/>
+		    <unit id="org.eclipse.lsp4j.sdk.feature.group" version="0.8.0.v20190703-1325"/>
```

Comment:
use `version="0.0.0"` as the p2 urls on the Jenkins server are not stable, they might disappear soon.

@kittaakos @spoenemann any ETA on the next 0.8.0 stable milestone or release?

Reply:
22.8 is the planned release-date for `0.8.0`.
https://projects.eclipse.org/projects/technology.lsp4j

Reply:
Changed to 0.0.0